src/encoded/static/components/browse/components/above-table-controls/SelectedFilesControls.js

```diff
@@ -28,16 +28,26 @@ export class SelectAllFilesButton extends React.PureComponent {
     static fieldsToRequest = [
         'accession',
         'produced_in_pub.display_title',
+        'lab.display_title',
 
         'processed_files.accession',
+        'processed_files.display_title',
```

Comment:
So we get proper title, ID, category, lab, etc. in "add to cart" and "checkout product" events when click Select All. Else might divide hits between "4DNFIAAAA1" and "4DNFIAAAA1.pairs.gz" cuz 1 case of tracking event uses accession and another uses display_title.